app/styles/_membership.less

```diff
@@ -31,7 +31,7 @@
     .col-name {
       min-width: 165px;
       padding: 0 5px 10px 0;
-      word-wrap: break-word;
+      word-break: break-all;
```

Comment:
Yeah, there are the `word-break` and `word-break-all` mixins. I'd prefer to use the former unless we really need `break-all`.